src/app.py

```diff
@@ -55,8 +52,16 @@ async def add_headers(request: Request, call_next: Callable) -> Response:
     return response
 
 
-@app.get("/api/download")
-def download(job_id: str, ordinal: str, file_path: str) -> FileResponse:
+@app.get("/api/download", tags=["stable"])
+def download(job_id: str, ordinal: int, file_path: str) -> FileResponse:
+    """
+    Downloads a file at the given `file_path`. This path should come from
```

Comment:
```suggestion
    Sends a file at the given `file_path`. This path should come from
```
From the server point of view this doesn't download, but actually upload. `Send` works for both sides.